extensions/data-transfer/portability-data-transfer-solid/src/main/java/org/datatransferproject/transfer/solid/SolidUtilities.java

```diff
@@ -0,0 +1,229 @@
+package org.datatransferproject.transfer.solid;
+
+import static com.google.common.base.Preconditions.checkState;
+
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpContent;
+import com.google.api.client.http.HttpHeaders;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpResponseException;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.common.collect.ImmutableList;
+import com.google.re2j.Pattern;
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.List;
+import java.util.function.Consumer;
+import org.apache.commons.io.IOUtils;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.vocabulary.RDF;
+import org.datatransferproject.transfer.solid.contacts.SolidContactsExport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SolidUtilities {
+  private static final Logger logger = LoggerFactory.getLogger(SolidContactsExport.class);
+  private static final Pattern PROBLEMATIC_TURTLE = Pattern.compile("(\\s\\d+\\.)\n");
+  private static final HttpTransport TRANSPORT = new NetHttpTransport();
+  private final String authCookie;
+  final HttpRequestFactory factory;
```

Comment:
Is this member exposed publicly for testing or for other reasons. We should annotate for it's use.